packages/ui/src/components/va-counter/VaCounter.vue

```diff
@@ -101,7 +101,7 @@ import { toRefs, computed, shallowRef, InputHTMLAttributes, ComputedRef, toRef,
 import omit from 'lodash/omit'
 import pick from 'lodash/pick'
 
-import { safeCSSLength } from '../../utils/css'
+import { safeCSSLength } from '@/utils/css'
```

Comment:
```suggestion
import { safeCSSLength } from '../../utils/css'
```

We use relative paths only.

Reply:
reverted changes